test/e2e/images_test.go

```diff
@@ -43,6 +43,17 @@ var _ = Describe("Podman images", func() {
 		Expect(session.LineInOuputStartsWith("docker.io/library/busybox")).To(BeTrue())
 	})
 
+	It("podman images with no images prints header", func() {
+		rmi := podmanTest.Podman([]string{"rmi", "-a"})
+		rmi.WaitWithDefaultTimeout()
+		Expect(rmi.ExitCode()).To(Equal(0))
+
+		session := podmanTest.Podman([]string{"images"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(0))
+		Expect(len(session.OutputToStringArray())).To(Equal(1))
```

Comment:
Some other tests are failing because they expect `.To(Equal(0))` which doesn't hold anymore.

Reply:
Replacing `-a` with `-aq` where I can - quiet output still prints nothing when 0 images are present